Dockerfile-ubuntu2204-dpdk2211

```diff
@@ -32,4 +32,4 @@ RUN wget http://fast.dpdk.org/rel/dpdk-${DPDK_VERSION}.tar.gz \
   && tar xvzf dpdk-${DPDK_VERSION}.tar.gz \
   && rm -rf dpdk-${DPDK_VERSION}.tar.gz \
   && mv dpdk-stable-${DPDK_VERSION} dpdk && cd dpdk \
-  && meson build && cd build && ninja && ninja install && ldconfig
```

Comment:
Maybe we should just deprecate KNI support in PcapPlusPlus for DPDK 22.11 and above?

Reply:
Ok